src/db.rs

```diff
@@ -314,12 +315,33 @@ impl KbsDb {
         let query_str = "SELECT kskeys FROM keysets WHERE keysetid = ?";
         let new_query_str = self.replace_binds(query_str);
 
-        let keyset_row = sqlx::query(&new_query_str)
+        // Need to check 2 cases: 1) no keyset keys are returned from the query; 2) keysets are
+        // returned but cannot be deserialized into JSON by serde_json::from_str().
+        let keyset_row = match sqlx::query(&new_query_str)
```

Comment:
Rather than using a match for each of these maybe we can use `.unwrap_or()` and put the logging inside the or. That might be more idiomatic. wdyt @dubek?

Reply:
IMO `.unwrap_or(x)` makes sense if "x" is a simple value. If you start packing several statements inside, then I think `match` statement is just fine.